apps/todo/angular/src/app/todo/todo-item/todo-item.component.html

```diff
@@ -1,11 +1,24 @@
-<todo-checkbox
+<!-- <todo-checkbox
```

Comment:
do not commit commented code

Reply:
fixed, it was late at night and it happened to me xd, anyway I did a --amend to not commit those changes

Reply:
perfect